dotcom-rendering/src/types/front.ts

```diff
@@ -254,6 +254,7 @@ export type DCRFrontCard = {
 	byline?: string;
 	showByline?: boolean;
 	avatarUrl?: string;
+	editionBrandings?: { edition: { id: EditionId } }[];
```

Comment:
I don't think we need this anymore? If I remember correctly, it was an idea we had at some point to check whether a container is labs container but it turns out there is no need for this check.